deployment/utils/remote-artifact-as-service.ts

```diff
@@ -117,7 +117,7 @@ export class RemoteArtifactAsServiceDeployment {
           volumeMounts,
           command:
             this.options.packageInfo.runtime === 'node'
-              ? ['/bin/sh', '-c', artifactUrl.apply(v => `yarn add ${v}`)]
+              ? ['/bin/sh', '-c', artifactUrl.apply(v => `pnpm add ${v}`)]
```

Comment:
```suggestion
              ? ['/bin/sh', '-c', artifactUrl.apply(v => `yarn add ${v}`)]
```

Reply:
Old code, pnpm is already there.

Reply:
https://github.com/kamilkisiela/graphql-hive/pull/477/commits/251d1c4ecf33c43bce896369be09291417f47590